lib/livebook/evaluator.ex

```diff
@@ -214,7 +216,15 @@ defmodule Livebook.Evaluator do
 
   def handle_cast({:handle_intellisense, send_to, ref, request, evaluation_ref}, state) do
```

Comment:
Is this function is used by both hover and ctrl+space?

Reply:
Correct, I normalized completion (ctrl+space), details (hover) and formatting into [intellisense request](https://github.com/livebook-dev/livebook/blob/f9ec058e43f53800e3e0af770352bebc08160c86/lib/livebook/runtime.ex#L33-L39). For formatting we [don't use evaluator](https://github.com/livebook-dev/livebook/blob/f9ec058e43f53800e3e0af770352bebc08160c86/lib/livebook/runtime/erl_dist/runtime_server.ex#L218) as it doesn't require evaluation context :)